new_examples/quickstart.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pandasai as pai
+
+os.environ["PANDASAI_API_URL"] = "http://localhost:8000/"
```

Comment:
@gdcsinaptik I'd remove this as it's not meant to be committed. The default value should be used by default